CSS/reset.css

```diff
@@ -4,7 +4,7 @@
 */
 
 html, body, div, span, applet, object, iframe,
-h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+/*h1,*/ h2, /*h3, h4,*/ h5, h6, /*p,*/ blockquote, pre,
 a, abbr, acronym, address, big, cite, code,
 del, dfn, em, img, ins, kbd, q, s, samp,
 small, strike, strong, sub, sup, tt, var,
```

Comment:
Please delete code you no longer need, it makes your files neater and easier to read! :smile:

Reply:
Deleted not useful code